examples/ssh/iperf3.json

```diff
@@ -0,0 +1,45 @@
+{
+    "logo": "",
+    "version": "1.0.0",
+    "name": "Iperf",
+    "description": "This script is designed to retrieve network speed data using the iperf3",
+    "category": "Software Integration",
+    "requirements": {
+        "credentials": true,
+        "sandbox_version": "1.12",
+        "others": [
+            "iperf3 installed",
+            "the server iperf3 must be running"
+        ]
+    },
+    "tags": [
+        "iperf3",
+        "server",
+        "network",
+        "test",
+        "Download speed",
+        "Upload speed",
+        "Download speed UDP",
+        "Upload speed UDP"
+    ],
+    "sample_period": 60000,
```

Comment:
The sample period value is in seconds

this is 16.66667 hours, was this intentional?